autodetect-dependabot.jl

```diff
@@ -0,0 +1,33 @@
+module AutodetectDependabot
+
+function get_possible_branch_names()
+    possible_branch_names = [
+        get(ENV, "APPVEYOR_PULL_REQUEST_HEAD_REPO_BRANCH", ""),
+        get(ENV, "APPVEYOR_REPO_BRANCH", ""),
+        get(ENV, "GITHUB_BASE_REF", ""),
+        get(ENV, "GITHUB_HEAD_REF", ""),
+        get(ENV, "GITHUB_REF", ""),
+        get(ENV, "TRAVIS_BRANCH", ""),
+        get(ENV, "TRAVIS_PULL_REQUEST_BRANCH", ""),
+    ]
+    return possible_branch_names
+end
+
+function chop_refs_head(branch_name::AbstractString)
+    if startswith(branch_name, "refs/heads/")
+        return chop(branch_name; head = 11, tail = 0)
```

Comment:
Could this be done with `replace` instead? I feel like that might be nicer than hard-coding the length.

Reply:
Done.